cognite/client/data_classes/agents/agents.py

```diff
@@ -134,14 +145,26 @@ def __init__(
         description: str | None = None,
         instructions: str | None = None,
         model: str | None = None,
+        labels: list[str] | None = None,
```

Comment:
@andeplane / @paco-cognite : This PR adds the following description to the docstring: "Always present in API responses", yet this read-class-definition says otherwise:

```py
        description: str | None = None,
        instructions: str | None = None,
        model: str | None = None,
        labels: list[str] | None = None,
```
The correct thing to do is to make them required (and drop the text in the docstring) 😄